caom2repo/caom2repo/core.py

```diff
@@ -89,13 +89,17 @@
 from six.moves.urllib.parse import urlparse
 
 # from . import version as caom2repo_version
-from . import version
+from caom2repo import version
 
 __all__ = ['CAOM2RepoClient']
 
 BATCH_SIZE = int(10000)
-SERVICE_URL = 'www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/' #TODO replace with SERVICE_URI when server supports it
 DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%f" #IVOA dateformat
+
+CAOM2REPO_FEATURE_ID = 'vos://cadc.nrc.ca~vospace/CADC/std/CAOM2Repository#obs-1.0'
```

Comment:
Not sure I understand what you are suggesting. Each application works with one or multiple feature IDs. In VOspace the feature IDs would be nodes, or transfer end points etc. The application interacts differently with each of the end points corresponding to different features, hence I don't know how can it be made generic.